Stratis.Bitcoin/Features/Consensus/ConsensusFeature.cs

```diff
@@ -13,227 +11,229 @@
 using Stratis.Bitcoin.Features.Consensus.CoinViews;
 using Stratis.Bitcoin.Features.Consensus.Deployments;
 using Stratis.Bitcoin.Utilities;
+using System;
+using System.Threading;
 
 namespace Stratis.Bitcoin.Features.Consensus
 {
-	public class ConsensusFeature : FullNodeFeature
-	{
-		private readonly DBreezeCoinView dBreezeCoinView;
-		private readonly Network network;
-		private readonly ConcurrentChain chain;
-		private readonly PowConsensusValidator consensusValidator;
-		private readonly LookaheadBlockPuller blockPuller;
-		private readonly CoinView coinView;
-		private readonly ChainState chainState;
-		private readonly IConnectionManager connectionManager;
-	    private readonly INodeLifetime nodeLifetime;
-	    private readonly Signals.Signals signals;
-		private readonly ConsensusLoop consensusLoop;
-		private readonly NodeSettings nodeSettings;
-	    private readonly NodeDeployments nodeDeployments;
-	    private readonly StakeChainStore stakeChain;
+    public class ConsensusFeature : FullNodeFeature
```

Comment:
Actually, what happened here was my VS untabified the file and re-ordered the usings ;)